test/integration/test_cpu_platform_profiles.c

```diff
@@ -0,0 +1,124 @@
+#include <rz_core.h>
```

Comment:
@thestr4ng3r I probably missed the addition of this dir. How are we going to split things that need to go in test/integration and test/unit? I believe there are already some things in unit that are not that "unit". WDYT?

Reply:
Integration tests in this case are the one that involve multiple parts/modules of Rizin, and usually represent some real use-case scenario.  They are good way to test possible use of the Rizin API.